test/storage/offline_database.test.cpp

```diff
@@ -990,7 +1000,9 @@ TEST(OfflineDatabase, TEST_REQUIRES_WRITE(ClearAmbientCache)) {
         db.clearAmbientCache();
     }
 
+#ifndef __QT__ // Qt doesn't decrease the size of the database file.
```

Comment:
> Qt doesn't decrease the size of the database file.

Should it?

Reply:
If it should, it's a Qt bug so out of scope. All commands are just passed-through to a Qt DB engine, there's no MapLibre implementation specifics.